Suggestion on the readme (#6)
OyaCanli/awesome-android-oem-issues: README.md

# "Awesome" Android OEM issues (Common problems with specific Android device types from OEMs)

> For us Android application developers, one of our worst nightmares is to deal with unexpected issues due to the variety of Original Equipment Manufacturers(OEMs). Issues usually reported by unhappy clients on the PlayStore, because we can't test our apps on every existing Android overlays.

Reviewer: ```suggestion
As Android application developers, one of our worst nightmares is to deal with unexpected issues due to the variety of Original Equipment Manufacturers(OEMs). Since we can't test our apps on every existing Android overlay, we usually recognize these late, through unhappy clients on the PlayStore.
```

> This repository has been created to share problems we encountered with specific brands and models, to be warned about them before starting a new feature and to discuss about solutions or workarounds.

Reviewer: ```suggestion
This repository has been created to share problems we encountered with specific brands and models, so that we can be aware of them before starting a new feature and to discuss solutions or workarounds.
```

## Browsing OEM Issues

> Issues reported by the community are listed [here](https://oyacanli.github.io/awesome-android-oem-issues/).
> 
> OEM issues are composed as GitHub discussions. You can [access them](https://github.com/OyaCanli/awesome-android-oem-issues/discussions) and start browsing through the issues. You can filter by the categories on the left, and you can use the search box if you look for something specific.

Reviewer: ```suggestion
Issues reported by the community are listed [on our website](https://oyacanli.github.io/awesome-android-oem-issues/).

Discussions for each issue are located in the discussions section of this repository. So alternatively, you can directly access them through [discussions](https://github.com/OyaCanli/awesome-android-oem-issues/discussions). You can filter by the categories on the left, and you can use the search box if you look for something specific.
```
